doc/api/perf_hooks.md

```diff
@@ -109,6 +109,18 @@ Creates a new `PerformanceMark` entry in the Performance Timeline. A
 `performanceEntry.duration` is always `0`. Performance marks are used
 to mark specific significant moments in the Performance Timeline.
 
+### performance.maxBufferSize
```

Comment:
By the description, I think the property name might be better as something more related?

Something like `maxEntries`? Unless it *needs* to expose the detail about being a buffer?

Reply:
I'm all for a better name. The current was chosen to have some resemblance to https://developer.mozilla.org/en-US/docs/Web/API/Performance/setResourceTimingBufferSize